fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -1043,6 +1039,7 @@ public InsertStmt generateInsertStmt(Set<String> materializedViewPartitions,
                 }
             }
         }
+        Tracers.close();
```

Comment:
Is there any possible to throw exception before close tracers?

Reply:
maybe, I move it